Binance.Net/BinanceBrokerageClient.cs

```diff
@@ -0,0 +1,758 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Binance.Net
+{
+    /// <summary>
+    /// Client providing access to the Binance Brokerage REST Api
+    /// </summary>
+    public class BinanceBrokerageClient : RestClient, IBinanceBrokerageClient
+    {
+        #region Fields
+        
+        private static BinanceBrokerageClientOptions defaultOptions = new BinanceBrokerageClientOptions();
+        private static BinanceBrokerageClientOptions DefaultOptions => defaultOptions.Copy();
+        
+        private readonly bool autoTimestamp;
+        private readonly TimeSpan autoTimestampRecalculationInterval;
+        private readonly TimeSpan timestampOffset;
+        private readonly TimeSpan defaultReceiveWindow;
+        
+        private double calculatedTimeOffset;
+        private bool timeSynced;
+        private DateTime lastTimeSync;
+        
+        private const string Api = "api";
+        private const string BrokerageApi = "sapi";
+        
+        private const string PublicVersion = "3";
+        private const string BrokerageVersion = "1";
+        
+        private const string CheckTimeEndpoint = "time";
+        private const string PingEndpoint = "ping";
+        
+        private const string SubAccountEndpoint = "broker/subAccount";
+        private const string BrokerAccountInfoEndpoint = "broker/info";
+        private const string EnableMarginForSubAccountEndpoint = "broker/subAccount/margin";
+        private const string EnableFuturesForSubAccountEndpoint = "broker/subAccount/futures";
+        private const string ApiKeySubAccountEndpoint = "broker/subAccountApi";
+        private const string ApiKeySubAccountPermissionEndpoint = "broker/subAccountApi/permission";
+        private const string ApiKeySubAccountCommissionEndpoint = "broker/subAccountApi/commission";
+        private const string ApiKeySubAccountCommissionFuturesEndpoint = "broker/subAccountApi/commission/futures";
+        private const string TransferEndpoint = "broker/transfer";
+        private const string RebatesRecentEndpoint = "broker/rebate/recentRecord";
+        private const string RebatesHistoryEndpoint = "broker/rebate/historicalRecord";
+        private const string EnableOrDisableBnbBurnForSubAccountSpotAndMarginEndpoint = "broker/subAccount/bnbBurn/spot";
+        private const string EnableOrDisableBnbBurnForSubAccountMarginInterestEndpoint = "broker/subAccount/bnbBurn/marginInterest";
+        private const string BnbBurnForSubAccountStatusEndpoint = "broker/subAccount/bnbBurn/status";
+
+        #endregion
+
+        #region Constructors
+        
+        /// <summary>
+        /// Create a new instance of BinanceBrokerageClient using credentials and the default options
+        /// </summary>
+        public BinanceBrokerageClient(ApiCredentials credentials)
```

Comment:
To keep in line with the other clients this constructor should have no parameters. (I understand this client won't work without credentials, but I'd like to keep it consistent).

Also, instead of calling the base directly it should call the other constructor with the default options. See BinanceClient constructors.